podcasts/Onboarding/Login/LoginLandingView.swift

```diff
@@ -0,0 +1,255 @@
+import SwiftUI
+
+struct LoginLandingView: View {
+    @EnvironmentObject var theme: Theme
+    @ObservedObject var coordinator: LoginCoordinator
+
+    var body: some View {
+        ProportionalValueFrameCalculator {
+            LoginLandingContent(coordinator: coordinator)
+        }
+    }
+
+    func configure(controller: UIViewController) {
+        let imageView = ThemeableImageView(frame: .zero)
+        imageView.imageNameFunc = AppTheme.pcLogoSmallHorizontalForBackgroundImageName
+        imageView.accessibilityLabel = L10n.accountLogin
+        controller.navigationItem.titleView = imageView
+
+        controller.navigationItem.rightBarButtonItem = UIBarButtonItem(title: "Not Now", style: .plain, target: coordinator, action: #selector(LoginCoordinator.dismissTapped))
+    }
+}
+
+private struct LoginLandingContent: View {
+    @EnvironmentObject var theme: Theme
+    @ObservedObject var coordinator: LoginCoordinator
```

Comment:
```suggestion
    var coordinator: LoginCoordinator
```

You don't need this to be an ObservedObject because it doesn't publish any changes.

Reply:
Done.